packages/server-wallet/src/wallet/store.ts

```diff
@@ -664,6 +669,10 @@ export class Store {
   async nextNonce(signingAddresses: Address[]): Promise<number> {
     return await Nonce.next(this.knex, signingAddresses);
   }
+
+  dbAdmin(): DBAdmin {
+    return new DBAdmin(this.knex);
+  }
```

Comment:
This is replicated in the `Wallet` class, I believe? Do we want to keep both?

Reply:
I think we need it here. Thought it wouldn't be too bad to keep it in both places for the time being.